physicsnemo/distributed/shard_utils/point_cloud_ops.py

```diff
@@ -456,42 +462,42 @@ def backward(
             Gradients for inputs (currently not implemented)
         """
 
-        raise MissingShardPatch("Backward pass for ring ball query not implemented.")
-
-        (
-            points1,
-            points2,
-            current_mapping,
-            current_num_neighbors,
-            current_outputs,
-        ) = ctx.saved_tensors
-
-        # We need to do a ring again in the backward direction.
-        # The backward pass is computed locally, and then the gradients
-        # and p2 are moved along the ring together.
-        # for i in range(world_size):
-        # Calculate which source rank this data is from
-
-        local_p2_grad = _ball_query_backward_primitive_(
-            points1[0],
-            points2[0],
-            current_mapping,
-            current_num_neighbors,
-            current_outputs,
-            mapping_grad,
-            num_neighbors_grad,
-            outputs_grad,
-        )
-        local_p1_grad = torch.zeros_like(points1)
-
-        return (
-            local_p1_grad,
-            local_p2_grad.unsqueeze(0),
-            None,
-            None,
-            None,
-            None,
-        )
+        # raise MissingShardPatch("Backward pass for ring ball query not implemented.")
```

Comment:
Should this be commented out? 

I'm actually surprised that Python isn't giving a syntax error here due to indentation without a `pass` here in the function body?

Reply:
Fixed good catch!